conformance_tests/tools/debug/src/test_debug.cpp

```diff
@@ -1085,6 +1085,11 @@ void zetDebugEventReadTest::run_read_events_in_separate_thread_test(
 
     LOG_INFO << "[Debugger] Sending interrupt from main thread";
     lzt::debug_interrupt(debugSession, device_thread);
+    std::vector<ze_device_thread_t> stopped_threads;
```

Comment:
This should be added to `read_and_verify_events_debugger_thread` instead I think